### PR TITLE
mark: fix pylint not-callable error on pytest.mark.parametrize(...), again

### LINKED ISSUE #7558: pytest 6: pylint still shows not-callable error for builtin pytest marks even after #7473 and #7476
I'm experiencing #7473 after the fix in #7476 with the following:

```python
import pytest

@pytest.mark.parametrize('a', [1, 2])
def test_a(a):
    pass
```

Where pylint still shows `not-callable` for `pytest.mark.parametrize`:

```
$ pylint -E test_pytest_7473.py
************* Module test_pytest_7473
test_pytest_7473.py:3:1: E1102: pytest.mark.parametrize is not callable (not-callable)
```

Environment:

- Python 3.8
- pytest==6.0.0
- pylint==2.5.3

FYI @bluetech

### PR DESCRIPTION
**Pull request by bluetech**
Fixes #7558, Refs #7473. Previous attempt: #7476.

Apparently the previous fix c1ca42b5c264b4ea1b4591bc12e0 didn't work. Hopefully this time I'm testing this correctly.

### CODE REVIEW

**Comment by bluetech** [comment]
Thanks for checking @The-Compiler. I'm more confident this one does the trick, so I'm merging and will do a backport.

File: src/_pytest/mark/structures.py [hunk 1]
```diff
@@ -473,14 +472,13 @@ def test_function():
...
-        xfail = cast(_XfailMarkDecorator, None)
-        parametrize = cast(_ParametrizeMarkDecorator, None)
-        usefixtures = cast(_UsefixturesMarkDecorator, None)
-        filterwarnings = cast(_FilterwarningsMarkDecorator, None)
+        skip = _SkipMarkDecorator(Mark("skip", (), {}))
+        skipif = _SkipifMarkDecorator(Mark("skipif", (), {}))
+        xfail = _XfailMarkDecorator(Mark("xfail", (), {}))
+        parametrize = _ParametrizeMarkDecorator(Mark("parametrize ", (), {}))
+        usefixtures = _UsefixturesMarkDecorator(Mark("usefixtures ", (), {}))
+        filterwarnings = _FilterwarningsMarkDecorator(Mark("filterwarnings ", (), {}))
```
Comment (graingert) [inline src/_pytest/mark/structures.py:481]: is there a reason these have an extra space? and skip/skipif/xfail do not?

Comment (graingert) [reply src/_pytest/mark/structures.py:481]: @bluetech ^

Comment (bluetech) [reply src/_pytest/mark/structures.py:481]: Oops, just a bad vim macro...

**Pull request merged**